scripts/st.d/73motr-di-corruption-detection

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

Comment:
Replace this line with
`# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates`